Assets/MixedRealityToolkit.SDK/Profiles/DefaultMixedRealityInputSystemProfile.asset

```diff
@@ -66,7 +66,7 @@ MonoBehaviour:
       reference: Microsoft.MixedReality.Toolkit.Input.InputSimulationService, Microsoft.MixedReality.Toolkit.Services.InputSimulation.Editor
     componentName: Input Simulation Service
     priority: 0
-    runtimePlatform: 16
+    runtimePlatform: 208
```

Comment:
Is this change intentional?

Reply:
Ah, yes it is: it's binary OR of Linux, Mac, Windows editors. (1 << 4) | (1 << 6) | (1 << 7). Thank you!